packages/vitest/src/node/stdin.ts

```diff
@@ -76,14 +76,20 @@ export function registerConsoleShortcuts(ctx: Vitest) {
     if (name === 'u')
       return ctx.updateSnapshot()
     // rerun all tests
-    if (name === 'a' || name === 'return')
+    if (name === 'a' || name === 'return') {
+      ctx.isFailedModel = false
```

Comment:
I don't think we should set properties of `ctx` outside it like this. Could this filter be toggled on/off inside the `core.ts` instead?

But your approach is right in the way that we indeed need to save the failure filter in state.